src/Builder/OpBuildTable.inc

```diff
@@ -40,7 +40,7 @@ op_dialect_version_map_["Compress"] = {11};
 op_dialect_version_map_["Concat"] = {13};
 op_dialect_version_map_["ConcatFromSequence"] = {11};
 op_dialect_version_map_["Constant"] = {19};
-op_dialect_version_map_["ConstantOfShape"] = {9};
+op_dialect_version_map_["ConstantOfShape"] = {20};
```

Comment:
I'm not sure exactly how this file works, but based on some of the other operators (like Clip above), should be `{20, 9}`?

Reply:
The {20, 9} would mean there are multiple versions of the operation implemented in onnxx-mlir, for example the interface of the operation changes between versions.  In the case of a `new data type` an additional version does need to be implemented, so `{20}` is correct.